packages/create-discord-bot/template/TypeScript/tsconfig.json

```diff
@@ -8,6 +8,7 @@
 		"moduleResolution": "NodeNext",
 		"target": "ESNext",
 		"outDir": "dist",
-		"skipLibCheck": true
+		"skipLibCheck": true,
+		"strictNullChecks": true
```

Comment:
Rather than enable this single option, please turn extends into an array and add `@sapphire/ts-config to it` that includes many options that might have been thought to be included in extra-strict